src/main/java/slimeknights/tconstruct/common/TinkerTags.java

```diff
@@ -126,11 +126,11 @@
     public static final INamedTag<EntityType<?>> SLIMES = forgeTag("slimes");
 
     private static INamedTag<EntityType<?>> tag(String name) {
-      return EntityTypeTags.func_232896_a_(Util.resource(name));
+      return EntityTypeTags.getTagById(Util.resource(name));
```

Comment:
We should switch to `createOptionalTag` for all our tags. New Forge feature that solves tag conflicts, see the `Tags` class for an example.

Reply:
Not finding a `Tags` class

Reply:
nvm, found it.

Reply:
Its in Forge